libs/src/services/hedgehog/index.js

```diff
@@ -86,6 +86,48 @@ class HedgehogWrapper {
       }
     }
 
+    /**
+     * Generate secure credentials to allow login
+     * @param {String} username username
+     */
+    hedgehog.getEntropyFromLocalStorage = async () => {
```

Comment:
@hareeshnagaraj there's a function that does this in walletManager https://github.com/AudiusProject/hedgehog/blob/master/src/walletManager.js#L62

Reply:
yes, however this is inaccessible at the upper level (account.js) and is required when generating the recovery link - unless I have misunderstood the access pattern for api layer

Reply:
@hareeshnagaraj there's two ways to deal with this

1. you basically need to call `WalletManager getEntropyFromLocalStorage()`. you could import `WalletManager` in account.js and call it like that, or attach `WalletManager` to the hedgehog object here.

2. you could just call the wallet manager from this file instead of redefining it and use this function as a wrapper for account.js like you do now

Reply:
i moved the link generation into hedgehogWrapper instead